build/dependencies.yaml

```diff
@@ -24,14 +24,6 @@ dependencies:
     - path: test/e2e/framework/nodes_util.go
       match: const etcdImage
 
-  - name: "docker"
```

Comment:
@neolit123 @Random-Liu - this removes tracking the docker version from k/k (this was going to happen though, since we are moving kubeadm out-of-tree)

FYI @justaugustus

Reply:
@yastij if we vendor the validatotors in k/k the deps.yaml logic should still work?

Reply:
we could by going through the `vendor/` folder, removed to discuss if we still want to track this or nott

Reply:
my vote goes +1 to track it because this will automate the release notes - e.g. "in 1.17 default version of docker is x.xxx"

Reply:
> in 1.17 default version of docker is x.xxx

what does it mean for the default version of docker to be 19.03? are kubeadm and dockershim the only places calling to docker directly? do we not track the version of the docker client that dockershim is using?

Reply:
@liggitt 
i don't think we do. perhaps @yastij and @dims can confirm.

Reply:
@liggitt - It is tracked under [go.mod](https://github.com/kubernetes/kubernetes/blob/master/go.mod#L244)